### PR TITLE
code comments and doc updates

### PR DESCRIPTION
**Pull request by jjnrmason**
Code comments and doc updates!
Make the doc smaller with the tip i shows you the other day.

### CODE REVIEW

File: game/game_logic/bank.py [hunk 1]
```diff
@@ -22,12 +22,14 @@ def bank_logic(player):
...
         print("------------------------------------------------------------------------------")
 
+        # Deposit coins
         if user_input == "D":
             user_input = ""
 
             while user_input != "A" and user_input != "C":
                 user_input = input("Do you want to deposit all your coins(A) or a certain amount(C)? ").upper()
 
+            # Deposit a cerain amount of coins.
```
Comment (johnmason27) [inline game/game_logic/bank.py:32]: spelling

**Review by jjnrmason — COMMENTED** [review]
1 spelling mistake found

File: game/game_logic/bank.py (hunk 1 above)
Comment (johnmason27) [inline game/game_logic/bank.py:32]:
```suggestion
            # Deposit a certain amount of coins.
```

**Comment by cBridges851** [comment]
Could put above line 27 in adventure_game.py (the new or returning player loop):
# Loops until the user specifies that they would like to create a new player or they are a returning player

File: game/adventure_game.py [hunk 2]
```diff
@@ -29,6 +32,7 @@ def adventure_game():
         player = Player()
         creating_user = True
 
+        # Make sure a user doesn't already exist.
         while creating_user:
             player.name = input("What is your first name? ")
             player_exists = FileLogic().add_new_player(PLAYER_FILENAME, player)
```
Comment (cBridges851) [inline game/adventure_game.py:38]:
```suggestion
            # Creates a new player or returns false if there is already a player with the name that the user inputted
            player_exists = FileLogic().add_new_player(PLAYER_FILENAME, player)
```

**Comment by cBridges851** [comment]
Above line 43 in adventure_game.py
# The user has been created, so the process of creating a user is complete
creating_user = False

**Comment by cBridges851** [comment]
Above line 73 adventure_game.py:
# Sees if the first inputted word is "GO" or "MOVE"

Then above line 74 in adventure_game.py:
# The direction for the user to move will be the second inputted word

**Comment by cBridges851** [comment]
Above line 13 in file_logic.py:
# Attempts to read the file specified in the file path

**Comment by cBridges851** [comment]
Above line 23:
# Deserialises the JSON

**Comment by cBridges851** [comment]
Above line 35 in file_logic.py:
\# Serialises the content that will be written to the file

**Comment by cBridges851** [comment]
Above line 56 in file_logic.py:
\# Creates a new player with the starting values

File: game/components/file_logic.py [hunk 3]
```diff
@@ -93,6 +94,7 @@ def retrieve_player(self, filepath, name):
         player = Player()
```
Comment (cBridges851) [inline game/components/file_logic.py:94]:
```suggestion
        # Creates a new instance of the player class
        player = Player()
```

File: game/components/file_logic.py [hunk 4]
```diff
@@ -93,6 +94,7 @@ def retrieve_player(self, filepath, name):
...
         playerFound = False
 
+        # Get a player and all it's profile contents.
```
Comment (cBridges851) [inline game/components/file_logic.py:97]:
```suggestion
        # Get a player and all its profile contents.
```

**Comment by cBridges851** [comment]
Above line 98 in file_logic.py:
\# Assigns the values from the file to the instance of the player class

**Comment by cBridges851** [comment]
Just noticed playerFound in file_logic.py - needs to be changed to player_found

**Comment by cBridges851** [comment]
Perhaps the print line in text-delay.py could be commented - why are sep, end and flush used.
Could also say about pausing before the next letter.

**Comment by cBridges851** [comment]
Above line 17 in bank.py:
\# Loops until the player wishes to leave the bank

**Review by 13JoshT — APPROVED** [review]

File: game/adventure_game.py (hunk 2 above)
Comment (cBridges851) [reply game/adventure_game.py:38]: Sorted on local

File: game/components/file_logic.py (hunk 3 above)
Comment (cBridges851) [reply game/components/file_logic.py:94]: Sorted on local

File: game/components/file_logic.py (hunk 4 above)
Comment (cBridges851) [reply game/components/file_logic.py:97]: Sorted on local

**Comment by cBridges851** [comment]
Will work on east next, will continue tomorrow.

**Review by cBridges851 — APPROVED** [review]

**Pull request merged**